docs/core/diagnostics/eventpipe.md

```diff
@@ -72,10 +72,11 @@ However, you can use the following environment variables to set up an EventPipe
 
 * `COMPlus_EventPipeOutputPath`: The path to the output EventPipe trace file when it's configured to run via `COMPlus_EnableEventPipe`. The default value is `trace.nettrace`, which will be created in the same directory that the app is running from.
 
-* `COMPlus_CircularBufferMB`: The size of the internal buffer that is used by EventPipe when it's configured to run via `COMPlus_EnableEventPipe`.
+* `COMPlus_EventPipeCircularMB`: A hexadecimal value that represents the size of EventPipe's internal buffer in megabytes. This configuration value is only used when EventPipe is configured to run via `COMPlus_EnableEventPipe`. The default buffer size is 1024MB which translates to this environment variable being set to `400`, since `0x400` == `1024`.
 
```

Comment:
Do we want to describe what happens when the value is too low or too high?

Reply:
That's still on my todo list. I figured I should do that in one PR that changes both this one and the dotnet-trace doc since the same issues will happen if values are too low/high.